legal-api/tests/unit/services/filings/validations/test_continuation_in.py

```diff
@@ -101,6 +99,8 @@ def test_invalid_party(mocker, app, session, legal_type):
     filing['filing']['header'] = {'name': 'continuationIn', 'date': '2019-04-08',
                                   'certifiedBy': 'full name', 'email': 'no_one@never.get', 'filingId': 1}
     filing['filing']['continuationIn'] = copy.deepcopy(CONTINUATION_IN)
+    filing['filing']['continuationIn']['isApproved'] = True
```

Comment:
This is the JSON data, right? Is there any risk of the UI setting this to True and controlling the execution?

Reply:
its used only for validation. if UI set it to true then reset of the properties will become mandatory (ie. office, parties, shares)

Reply:
Does this expose the risk of the UI filing a continuation application without an actual review?

Reply:
No its not